src/backend.py

```diff
@@ -589,34 +588,43 @@ def app_setScoreCap(request):
     json_data = request.data
     if "on-machine" in json_data:
         info = ds_read_record("extras", 0)
-        info["other"] = json_data["on-machine"]
+        info["enter_intials_on_game"] = bool(json_data["on-machine"])        
         ds_write_record("extras", info, 0)
 
 
 @add_route("/api/settings/tournament_mode")
-def app_getTournamentMode(request):
-    import SharedState
-
-    return json_dumps({"tournament_mode": SharedState.tournamentModeOn}), 200
+def app_getTournamentMode(request):    
+    tournament_mode = ds_read_record("extras", 0)["tournament_mode"]
+    return json_dumps({"tournament_mode": tournament_mode}), 200
 
 
 @add_route("/api/settings/tournament_mode", method="POST", auth=True)
 def app_setTournamentMode(request):
-    import SharedState
-
-    SharedState.tournamentModeOn = int(request.data["tournament_mode"])
-
+    json_data = request.data
+    if "tournament_mode" in json_data:
+        info = ds_read_record("extras", 0)
+        info["tournament_mode"] = bool(json_data["tournament_mode"])
+        ds_write_record("extras", info, 0)
```

Comment:
```suggestion
    info = ds_read_record("extras", 0)
    info["tournament_mode"] = bool(request.data["tournament_mode"])
    ds_write_record("extras", info, 0)
```

if tournament mode isn't there the request will fail, which is what we want so the server will send a error to the web UI with some info

Reply:
I expect we will add other switches - like display ip address on display.  not sure how you want to organize them - all in one route or route for each?

Reply:
Yeah that's true. We would want them all in one router eventually. This is fine as is then.